python/py-pyaudio/Portfile

```diff
@@ -49,9 +50,9 @@ if {${name} ne ${subport}} {
         xinstall -d ${destroot}${prefix}/share/doc/py${python.version}-pyaudio
         xinstall -m 640 {*}[glob ${worksrcpath}/examples/*] \
             ${destroot}${prefix}/share/doc/py${python.version}-pyaudio
-        xinstall -d ${destroot}${prefix}/share/examples/py-pyaudio
+        xinstall -d ${destroot}${prefix}/share/examples/py${python.version}-pyaudio
         xinstall -m 640 {*}[glob ${worksrcpath}/examples/*] \
-            ${destroot}${prefix}/share/examples/py-pyaudio
+            ${destroot}${prefix}/share/examples/py${python.version}-pyaudio
     }
```

Comment:
it seems redundant to install the files ```/examples/*``` twice, both to ```/share/doc/py${python.version}-pyaudio``` and ```/share/examples/py${python.version}-pyaudio```; please pick one

Reply:
thank you